mp-backend/db-tables.yml

```diff
@@ -22,6 +22,33 @@ authTable:
       ProvisionedThroughput:
           ReadCapacityUnits: 1
           WriteCapacityUnits: 1
+    ProvisionedThroughput:
+      ReadCapacityUnits: 1
+      WriteCapacityUnits: 1
+recipeTable:
+  Type: AWS::DynamoDB::Table
+  Properties:
+    TableName: recipe
+    AttributeDefinitions:
+      - AttributeName: id
+        AttributeType: S
+    KeySchema:
+      - AttributeName: id
+        KeyType: HASH
```

Comment:
We should add a sort key that is the UserId, with type string.

A partition key can be defined in two ways:
(1) Just a partition key (recipeId)
(2) A partition key and a sort key (reipceId and userId)

With option (2) we are able to sort through all the recipes for the user that owns them.